php/Dockerfile

```diff
@@ -28,9 +37,16 @@ COPY --from=clearlinux/os-core:latest / /os_core_install/
 RUN cd / && \
     find os_core_install | sed -e 's/os_core_install/install_root/' | xargs rm -d &> /dev/null || true
 
-
 FROM clearlinux/os-core:latest
 
+ENV OPCACHE_REVALIDATE_FREQ=0 \
+    OPCACHE_VALIDATE_TIMESTAMPS=0 \
+    OPCACHE_ENABLE_CLI=1 \
+    OPCACHE_OPTIMIZATION_LEVEL=0x7FFFFFFF \
```

Comment:
not sure if this 0x7FFFFFFF is best configuration optimization level? Do we need additional effort to figure out the best one? 
https://gist.github.com/ck-on/4959032?ocp.php